EX1/HW5.py

```diff
@@ -0,0 +1,118 @@
+import os
+import datetime
+import random
+import re
+
```

Comment:
Documantation :

“Code tells you how; Comments tell you why.”

— [Jeff Atwood]

https://realpython.com/documenting-python-code/#commenting-vs-documenting-code